pytest/common.py

```diff
@@ -1,6 +1,42 @@
 import signal
 from includes import *
 from threading import Thread
+from RLTest import Env, Defaults
+import inspect
+import unittest
+
+class Colors(object):
```

Comment:
Maybe you want this library: https://pypi.org/project/colored/

Reply:
Lets do it on separate PR as RLTest also uses it.